### ISSUE TITLE
Changing language shows unload confirmation dialog multiple times

### ISSUE THREAD

**Issue by RaenonX** [RaenonX-DL/dragalia-site-front#279]
Caused by the line below originally to handle `<NextLink>`.

https://github.com/RaenonX-DL/dragalia-site-front/blob/2f96b554b8192ade328fa5d15f6c90f7feb45505/src/i18n/switch.tsx#L25

After #254, this line is no longer needed.
